the-blue-alliance-ios/View Controllers/Match/MatchQueryOptionsViewController.swift

```diff
@@ -0,0 +1,166 @@
+import CoreData
+import Foundation
+import MyTBAKit
+import TBAKit
+import UIKit
+
+protocol MatchQueryOptionsDelegate: AnyObject {
+    func updateQuery(query: MatchQueryOptions)
+}
+
+// Backing enums to setup our table view data source
+private enum QuerySections: String, CaseIterable {
+    case sort = "Sort"
+    case filter = "Filter"
+}
+
+private enum SortRows: CaseIterable {
+    case reverse
+}
+
+private enum FilterRows: CaseIterable {
+    case favorites
+}
+
+private protocol QueryableOptions {
+    static func defaultQuery() -> Self
+
+    var isDefault: Bool { get }
+}
+
+// Backing structs to power our data
+struct MatchQueryOptions: QueryableOptions {
+
+    var sort: MatchSortOptions
+    var filter: MatchFilterOptions
+
+    static func defaultQuery() -> MatchQueryOptions {
+        return MatchQueryOptions(sort: MatchQueryOptions.MatchSortOptions.defaultQuery(), filter: MatchQueryOptions.MatchFilterOptions.defaultQuery())
+    }
+
+    struct MatchSortOptions: QueryableOptions {
+        var reverse: Bool
+
+        static func defaultQuery() -> MatchSortOptions {
+            return MatchSortOptions(reverse: false)
+        }
+
+        var isDefault: Bool {
+            return reverse == false
+        }
+    }
+
+    struct MatchFilterOptions: QueryableOptions {
+        var favorites: Bool
+
+        static func defaultQuery() -> MatchFilterOptions {
+            return MatchFilterOptions(favorites: false)
+        }
+
+        var isDefault: Bool {
+            return favorites == false
+        }
+    }
+
+    var isDefault: Bool {
+        return sort.isDefault && filter.isDefault
+    }
+}
+
+class MatchQueryOptionsViewController: TBATableViewController {
+
+    private var myTBA: MyTBA
+    private var query: MatchQueryOptions
+
+    weak var delegate: MatchQueryOptionsDelegate?
+
+    init(query: MatchQueryOptions, myTBA: MyTBA, persistentContainer: NSPersistentContainer, tbaKit: TBAKit, userDefaults: UserDefaults) {
+        self.query = query
+        self.myTBA = myTBA
+
+        super.init(style: .plain, persistentContainer: persistentContainer, tbaKit: tbaKit, userDefaults: userDefaults)
```

Comment:
`grouped` might be better here

Reply:
or, this is a new iOS 13 feature, we could look to use the `insetGrouped` (which we could make the default everywhere)